stable/moodle/README.md

```diff
@@ -92,9 +104,55 @@ $ helm install --name my-release -f values.yaml stable/moodle
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+### Ingress without TLS
+For using ingress (example without tls):
+```console
+$ helm install --name my-release \
+  --set ingress.enabled=True,ingress.hosts[0]=moodle.domain.com,moodleUsername=admin,moodlePassword=password,mariadb.mariadbRootPassword=secretpassword,serviceType=ClusterIP stable/moodle
```

Comment:
I suggest only defining `ingress.enabled=True` and `ingress.hosts[0]=moodle.domain.com` in this command example to demonstrate the required parameters to enable ingress.